.prettierignore

```diff
@@ -0,0 +1,14 @@
+.cache/
+**/corrupted-package/package.json
+**/corrupted.json
+**/invalid.json
+**/verdaccio-corrupted.db.json
+**/wrong.package.json
+**/invalid.js
+/website/.cache/
+/website/crowdin/
+/website/public/
+/website/src/dictionaries/
+CHANGELOG.md
+CONTRIBUTORS.md
+node_modules/
```

Comment:
I'd ignore storage folders, those are generated and might change while run local test.

Reply:
Do you mean `test-storage` ?

Reply:
yes, but ignore my comment, let's see how it reacts.